pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -405,3 +422,34 @@ func fixCrcDnsmasqConfigFile() (bool, error) {
 
 	return true, nil
 }
+func checkCrcNetworkManagerConfig() (bool, error) {
+	c := []byte(crcNetworkManagerConfig)
+	_, err := os.Stat(crcNetworkManagerConfigPath)
+	if err != nil {
+		return false, fmt.Errorf("File not found: %s: %s", crcNetworkManagerConfigPath, err.Error())
+	}
+	config, err := ioutil.ReadFile(crcNetworkManagerConfigPath)
+	if err != nil {
+		return false, fmt.Errorf("Error opening file: %s: %s", crcNetworkManagerConfigPath, err.Error())
+	}
+	if !bytes.Equal(config, c) {
+		return false, fmt.Errorf("Config file contains changes: %s", crcNetworkManagerConfigPath)
+	}
+	return true, nil
+}
+func fixCrcNetworkManagerConfig() (bool, error) {
+	cmd := exec.Command("sudo", "tee", crcNetworkManagerConfigPath)
```

Comment:
Why not using `RunWithPrivilege` ?

Reply:
`RunWithPrivilege` is limited, i cannot send `stdIn` to it atm.

Reply:
created a new func `RunWithDefaultLocale` and used it in all the places where we are comparing/checking o/p from commands!